server/calls.go

```diff
@@ -52,12 +52,20 @@ type videoCall struct {
 	parties map[*Session]callPartyData
 	// Call message seq ID.
 	seq int
+	// Call message content.
+	content interface{}
+	// Time when the call was accepted.
+	acceptedAt time.Time
 }
 
-func (call *videoCall) messageHead() map[string]interface{} {
+func (call *videoCall) messageHead(newState string, duration int64) map[string]interface{} {
```

Comment:
`int64` looks like an overkill. `int` should be plenty enough.

Reply:
Changed.